airflow/plugins_manager.py

```diff
@@ -84,12 +84,16 @@ def load_entrypoint_plugins(entry_points, airflow_plugins):
     :type airflow_plugins: List[AirflowPlugin]
     :return: List[Type[AirflowPlugin]]
     """
+    from airflow.www.app import  app
     for entry_point in entry_points:
         log.debug('Importing entry_point plugin %s', entry_point.name)
         plugin_obj = entry_point.load()
         if is_valid_plugin(plugin_obj, airflow_plugins):
             if callable(getattr(plugin_obj, 'on_load', None)):
-                plugin_obj.on_load()
+                plugin_obj.on_load(
+                    flask_app=app,
+                    configuration=configuration
```

Comment:
Does this count as a breaking change? (I can't remember when we added this method.)

Reply:
The plugin class on_load has *args, and **kwargs by default, so this shouldn't break.

Reply:
Resolved in https://github.com/apache/airflow/pull/4706